run_agents.py

```diff
@@ -1,21 +0,0 @@
-import numpy as np
-import matplotlib.pyplot as plt
-
-import maze_runner as mr
-
-
-# Initialize MazeEnvironment
-env = mr.MazeEnvironment(250)
-#env.print_maze()
```

Comment:
Argparse would allow you to control printing with a command line argument instead of commenting and uncommenting this line